cucumber-report/html-report/formatter.js

```diff
@@ -0,0 +1,228 @@
+var CucumberHTML = {};
```

Comment:
Same for cucumber-report directory.
Only code and mandatory libs should be in repository.